uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/document-list.controller.spec.js

```diff
@@ -344,6 +344,41 @@ define([
       });
     });
 
+    describe('document is saved', function () {
+      var document, changedDoc;
+
+      beforeEach(function () {
+        initController();
+      });
+
+      describe('when adding new document', function () {
+        beforeEach(function () {
+          $rootScope.$broadcast('documentFormSuccess', documentFabricator.single(), {});
+          $scope.$apply();
+
+          document = controller.list[controller.list.length - 1];
+        });
+
+        it('adds new document in the document list', () => {
+          expect(document).toEqual(documentFabricator.single());
+        });
+      });
+
+      describe('when editing existing document', function () {
+        beforeEach(function () {
+          changedDoc = angular.extend({}, documentFabricator.list()[0], {status_id: '4', activity_type_id: '70'});
```

Comment:
```javascript
changedDoc = angular.extend(
  {}, 
  documentFabricator.list()[0], 
  { status_id: '4', activity_type_id: '70' });
```
this will break the long line and increase readability